cupy_backends/hip/cupy_hipblas.h

```diff
@@ -342,6 +342,22 @@ cublasStatus_t cublasZgerc(cublasHandle_t handle, int m, int n, cuDoubleComplex*
                         reinterpret_cast<hipblasDoubleComplex*>(A), lda);
 }
 
+cublasStatus_t cublasSsbmv(cublasHandle_t handle, cublasFillMode_t uplo, int n, int k,
+                           const float* alpha, const float* A, int lda,
+                           const float* x, int incx,
+                           const float* beta, float* y, int incy) {
+    return hipblasSsbmv(handle, convert_hipblasFillMode_t(uplo), convert_hipblasOperation_t(trans),
```

Comment:
```suggestion
    return hipblasSsbmv(handle, convert_hipblasFillMode_t(uplo),
```

Reply:
Those changes have been concurrently implemented. I cannot change the status of the issue. It's still in "changes requested"